fpdcc/ccfp-asset-dashboard
# Setup React with a basic view and model for UI development

## Overview
This PR integrates React with a basic model and view. The model is based on the `~/raw/simplified.csv` data. The PR includes a management script for loading that data in for local development. 

The PR is part of the work for issue #1

### Notes
This is my first time working with Datamade's React/Django setup. Am I correctly [serializing the QuerySet](https://github.com/datamade/ccfp-asset-dashboard/blob/23dd0d81d7e10fd84ac2ddfda2542b14d1bf8bf0/asset_dashboard/views.py#L14)? The React component's props [receive that object as a string](https://github.com/datamade/ccfp-asset-dashboard/blob/23dd0d81d7e10fd84ac2ddfda2542b14d1bf8bf0/asset_dashboard/static/js/pages/index.js#L6) -- should it already be a json object at that point?

I didn't create a new Django app module or put much effort in the app's view/url design or write any tests, since this is a UI oriented task and that code is likely to change. Let me know if that's a mistaken assumption.

## Testing Instructions
- Clone the repo and checkout the branch: `git clone git@github.com:datamade/ccfp-asset-dashboard.git && git fetch origin sammc/basic-model-view && git checkout sammc/basic-model-view`
- Follow the build directions in the README. 
- Visit localhost:8000 and confirm that the data renders on the screen, under the big text that says `Django-React Integration`.
